docs/standard/base-types/best-practices-strings.md

```diff
@@ -135,7 +135,7 @@ Specifying the <xref:System.StringComparison.Ordinal?displayProperty=nameWithTyp
 
 Ordinal comparisons are string comparisons in which each byte of each string is compared without linguistic interpretation; for example, "windows" doesn't match "Windows". This is essentially a call to the C runtime `strcmp` function. Use this comparison when the context dictates that strings should be matched exactly or demands conservative matching policy. Additionally, ordinal comparison is the fastest comparison operation because it applies no linguistic rules when determining a result.
 
-Strings in .NET can contain embedded null characters. One of the clearest differences between ordinal and culture-sensitive comparison (including comparisons that use the invariant culture) concerns the handling of embedded null characters in a string. These characters are ignored when you use the <xref:System.String.Compare%2A?displayProperty=nameWithType> and <xref:System.String.Equals%2A?displayProperty=nameWithType> methods to perform culture-sensitive comparisons (including comparisons that use the invariant culture). As a result, in culture-sensitive comparisons, strings that contain embedded null characters can be considered equal to strings that don't.
+Strings in .NET can contain embedded null characters (and other non-printing characters). One of the clearest differences between ordinal and culture-sensitive comparison (including comparisons that use the invariant culture) concerns the handling of embedded null characters in a string. These characters are ignored when you use the <xref:System.String.Compare%2A?displayProperty=nameWithType> and <xref:System.String.Equals%2A?displayProperty=nameWithType> methods to perform culture-sensitive comparisons (including comparisons that use the invariant culture). As a result, in culture-sensitive comparisons (including in the invariant culture), strings that contain embedded null characters can be considered equal to strings that don't, and embedded non-printing characters might be skipped for the purpose of <xref:System.String.StartsWith%2A?displayProperty=nameWithType>.
```

Comment:
This fact was previously entirely undocumented, as far as I can tell. I added the "including in the invariant culture" because if you're used to dealing with the invariant culture primarily in the context of `DateTime`, you might not think of the words "culture-sensitive" as including it.

Reply:
How is this?

- Removed the wording around `As a result, in culture-sensitive comparisons (including in the invariant culture)` as it's the third time `(including....)` is used and the context of the sentence already establishes we're talking about culture-sensitive comparisons.

- Clarified that it's not just `StartsWith`, but other related methods (much like your suggestion above)

```suggestion
Strings in .NET can contain embedded null characters (and other non-printing characters). One of the clearest differences between ordinal and culture-sensitive comparison (including comparisons that use the invariant culture) concerns the handling of embedded null characters in a string. These characters are ignored when you use the <xref:System.String.Compare%2A?displayProperty=nameWithType> and <xref:System.String.Equals%2A?displayProperty=nameWithType> methods to perform culture-sensitive comparisons (including comparisons that use the invariant culture). As a result, strings that contain embedded null characters can be considered equal to strings that don't. Embedded non-printing characters might be skipped for the purpose of string comparison methods, such as <xref:System.String.StartsWith%2A?displayProperty=nameWithType>.
```